src/lib/components/ScriptureViewSofria.svelte

```diff
@@ -374,11 +381,11 @@ TODO:
                         description: 'Set up; Book heading',
                         test: () => true,
                         action: ({ context, workspace }) => {
-                            /* console.log(
-                                'Start Document: %o, %o',
-                                context,
-                                context.document.metadata.document
-                            );*/
+                            // console.log(
+                            //     'Start Document: %o, %o',
+                            //     context,
+                            //     context.document.metadata.document
+                            // );
```

Comment:
keep the commented out console.log?

Reply:
Yes

Reply:
Changing it from a /* */ type to one that the command / will take out, now that I know about that command.  I am uncommenting all of these every time I go to do some change to understand what is coming in and out.